pymeasure/instruments/anaheimautomation/dpseriesmotorcontroller.py

```diff
@@ -330,7 +331,7 @@ def write(self, command):
         super().write(cmd_str)
 
     def wait_for_completion(self, interval=0.5):
-        """ Block until the controller is not "busy" (i.e. block until the motor is no longer moving.)
+        """ Block until the controller is not "busy" (i.e. block until the motor is no longer moving.)  # noqa: E501
```

Comment:
AFAICT, this makes `  # noqa: E501` part of the docstring, correct?
Given that `noqa is supposed to be a source _comment_, does it also work if given at the end of the multiline string (i.e. `"""  # noqa: E501`?

Reply:
That is actually a very good point (that I should've thought about to check, woops), and you're right, the `noqa` does appear in the docstring; it does indeed also work if I place it after the string, so I corrected it.